CIP-1694/README.md

```diff
@@ -296,20 +296,27 @@ can also be varied by the governance action.
 This gives a great deal of flexibility to the composition of the committee.
 In particular, it is possible to elect an empty committee if the community wishes to abolish the constitutional committee entirely. Note that this is different from a state of no-confidence and still constitutes a governance system capable of enacting proposals.
 
+There will be a new protocol parameter for the minimal size of the committee,
+itself a nonnegative number.
+
 #### Term limits
 
-Each newly elected constitutional committee will have a term limit.
-The system will automatically enter a state of no-confidence when the term limit for the constitutional
-committee expires.  This limit is a governance protocol parameter, which specifies the maximum number of epochs
-during which the committee can ratify governance actions.  When the committee term limit expires, all governance
-actions will be **dropped** and the protocol enters a state of no confidence.
-This means that the committee should plan for its own replacement if it wishes to avoid disruption.
+Each newly elected constitutional committee will have per-member term limits.
+Per-member limits allow for a rotation scheme, such as a third of the committee
+expiring every year.
+Expired members can no longer vote.
+Member can also willingly resign early, which will be marked on-chain as an expired member.
 
-The term limit will reset whenever the  "New constitutional committee and/or threshold" governance action is enacted, even if the same committee is re-elected
-and the threshold remains unchanged.  This allows Ada holders to confirm their confidence in the committee if they wish.
-Note that the term limit is calculated at the point the committee is elected.  Any change in the underlying protocol
-parameter only affects the term that applies to future committees, and does not change the term of the current committee.
+The system will automatically enter a state of no-confidence when the number of non-expired
+committee members falls below the minimal size of the committee.
+For example, a committee of size five with a quorum of three and two expired members can still
```

Comment:
I believe it is essential to implement the proposed changes, as they appear to be reasonable and beneficial. In particular, I would like to emphasize the importance of requiring a unanimous "yes" vote from all currently active CC members when altering the quorum size. This is crucial for several reasons:

1. **Ensuring Consensus:** A unanimous vote guarantees that all committee members are in agreement, fostering a collective decision-making process that encourages collaboration and harmony within the group.
2. **Upholding Integrity:** By requiring the approval of all active members, it becomes more difficult for a minority to manipulate or unduly influence the decision-making process. This helps maintain the integrity and fairness of the committee's actions.
3. **Mitigating Potential Conflicts:** Unanimous agreement on such a critical matter minimizes the potential for conflicts or disagreements arising from differing opinions or perspectives. This ultimately leads to a more cohesive and effective committee.

I am uncertain if the MVG model allows for a unanimous voting requirement when altering the quorum size. However, I believe it is vital to explore this possibility to ensure that important decisions, such as adjusting the quorum threshold, are made by all active CC members in unison.